docs/tutorials/writing_custom_configaudit_policies.md

```diff
@@ -0,0 +1,218 @@
+# Writing Custom Configuration Audit Policies
+
+Starboard ships with a set of pre-installed configuration audit policies defined as OPA [Rego] policies. You can also
```

Comment:
Where can people learn more about our pre-installed configuration audit policies? Would be worth linking

Reply:
There's no such place, but I'm going to list built-in policies in scope of #1019. Then we could update this tutorial with reference.